lang/en.json

```diff
@@ -275,10 +275,6 @@
     "defaultMessage": "The name should be all lowercase",
     "description": "Warning text for new Python file with uppercase"
   },
-  "file-name-not-blank": {
```

Comment:
This was a project name specific string but I think was unintentionally reprefixed along with the Create file related strings in the rejig of the validation for that dialog.